linux_os/guide/system/network/network-wireless/wireless_software/wireless_disable_interfaces/ansible/shared.yml

```diff
@@ -1,3 +1,9 @@
+{{% if platform_package_overrides.NetworkManager %}}
+{{% set network_manager_package_name = platform_package_overrides.NetworkManager %}}
```

Comment:
I'm in doubt if we should use the platform_package_overrides here or just be explicit for each of the platforms directly.
@Mab879  @jan-cerny  any opinions about it?

Reply:
@dodys I went through and changed this since you said the platform_package_overrides is mainly used in the test suite.